src/Workspaces/MSBuildWorkspaceLoader.cs

```diff
@@ -51,6 +51,9 @@ internal static class MSBuildWorkspaceLoader
             {
                 try
                 {
+                    // In the case of loading a specific project, we don't want to load referenced projects, which is very time consuming, when there is already
+                    // a build on disk we can pull metadata from.
+                    workspace.LoadMetadataForReferencedProjects = true;
```

Comment:
I assumed this wouldn't be useful in the solution loading case, since referenced projects would likely be part of the solution. Thinking about it again, it isn't strictly necessary, so this may just need to be enabled generally.